src/test/cli/rbd/help.t

```diff
@@ -610,22 +611,29 @@
   
   rbd help device detach
   usage: rbd device detach [--device-type <device-type>] [--pool <pool>] 
-                           [--image <image>] [--snap <snap>] 
+                           [--namespace <namespace>] [--image <image>] 
+                           [--snap <snap>] [--snap-id <snap-id>] 
                            [--options <options>] 
-                           <image-or-snap-or-device-spec> 
+                           <image-or-snap-or-device-spec> <snap-spec> 
```

Comment:
This is wrong: `rbd device detach` takes a single positional argument, nothing changes with the addition of the `--snap-id` option.